nncf/quantization/algorithms/fast_bias_correction/backend.py

```diff
@@ -156,13 +156,23 @@ def create_blob(shape: Tuple[int], data: List[float]) -> np.ndarray:
 
     @staticmethod
     @abstractmethod
-    def get_initializer_value(model: TModel, initializer_name: str) -> np.ndarray:
+    def get_bias_value(model: TModel, node: NNCFNode) -> np.ndarray:
         """
-        Returns initializer value in the NumPy format.
+        Returns bias value in the NumPy format of provided node.
 
         :param model: Backend-specific model for the initializer finding.
-        :param initializer_name: Name of the tensor/initializer to find in the model.
-        :return: Initializer value in the NumPy format.
+        :param node: Node of NNCFGraph with bias value.
+        :return: Bias value in the NumPy format.
+        """
+    @staticmethod
+    @abstractmethod
+    def is_quantized_weights(node: NNCFNode, model: TModel) -> bool:
+        """
+        Checks whether the node is quantised or not
```

Comment:
`quantized`?

Reply:
Done

> https://github.com/openvinotoolkit/nncf/blob/e2e904f75f84525dc23064d4c68a8010155cbc57/nncf/quantization/algorithms/min_max/algorithm.py#L247-L251
> 
> It would be more common to change `edge_name` to `port_id` in `TargetPoint`.

Done

Reply:
Done